src/helpers/git_helpers/remote_helpers.sh

```diff
@@ -3,7 +3,12 @@
 
 # Returns the url for the remote
 function remote_url {
-  git remote -v | grep "origin.*fetch" | awk '{print $2}'
+  mock_remote=$(git config --get git-town.testing.remote-url)
```

Comment:
I really dislike this but don't see another way to accomplish this. Can we at least only check this when running tests by setting an environment variable in our tests?